static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss

```diff
@@ -0,0 +1,83 @@
+.dropdown-menu-fallback:checked {
+
+    & ~ .dropdown-menu {
+        display: block;
+    }
+}
+
+.dropdown-menu {
+    display: none;
+    position: absolute;
+    top: 29px; //visually this sits right under the text
+    left: -($gs-gutter / 2);
+    width: $gs-column-width * 3;
+    background-color: #ffffff;
+    border-radius: 3px;
+    padding: $gs-baseline / 2 0;
+    margin: 0;
+    box-shadow: 0 0 5px 0 rgba(0, 0, 0, .25);
+
+    // triangle
+    &:before,
+    &:after {
+        content: '';
+        position: absolute;
+    }
+
+    &:before {
+        top: -( $gs-baseline / 2 + 1);
+        left: $gs-gutter - 2;
+        border-bottom: $gs-baseline / 2 + 1 solid $neutral-5;
+        border-left: $gs-baseline / 2 + 1 solid transparent;
+        border-right: $gs-baseline / 2 + 1 solid transparent;
+    }
+
+    &:after {
+        top: -( $gs-baseline / 2);
+        left: $gs-gutter - 1;
+        border-bottom: $gs-baseline / 2 solid #ffffff;
+        border-left: $gs-baseline / 2 solid transparent;
+        border-right: $gs-baseline / 2 solid transparent;
+    }
+}
+
+.dropdown-menu--open {
+    display: block;
+}
+
+.dropdown-menu__title {
+    @include fs-textSans(2);
+    color: $neutral-1;
+    cursor: pointer;
+    display: block;
+    outline: none;
+    padding: $gs-baseline + 1 $gs-gutter $gs-baseline;
+    position: relative;
+    box-sizing: border-box;
+    margin-top: -1px;
```

Comment:
Since this is `position: relative` you could use `top: -1px` instead, because it's used to positioning instead of modeling margins ...